### PR TITLE
Enhancement completed there should be tooltip text on mouse hover on compare icon on product #3550

### PR DESCRIPTION
**Pull request by devansh-webkul**
**ENHANCEMENT:**
Link: https://github.com/bagisto/bagisto/issues/3550

### CODE REVIEW

File: packages/Webkul/Velocity/src/Resources/views/shop/products/wishlist.blade.php [hunk 1]
```diff
@@ -37,7 +37,9 @@ class="unset wishlist-icon {{ $addWishlistClass ?? '' }} text-right"
             add-class="{{ $addWishlistClass ?? '' }}"
             move-to-wishlist="{{ $isMoveToWishlist ?? null}}"
             added-text="{{ __('shop::app.customer.account.wishlist.add') }}"
-            remove-text="{{ __('shop::app.customer.account.wishlist.remove') }}">
+            remove-text="{{ __('shop::app.customer.account.wishlist.remove') }}"
+            add-tooltip="{{ __('velocity::app.shop.wishlist.add-wishlist-text') }}"
+            remove-tooltip="Remove from wish list">
```
Comment (rahulcs0082) [inline packages/Webkul/Velocity/src/Resources/views/shop/products/wishlist.blade.php:42]: Add translations.

Comment (devansh-webkul) [reply packages/Webkul/Velocity/src/Resources/views/shop/products/wishlist.blade.php]: Added

**Comment by rahulcs0082** [comment]
Hi @devansh-webkul 

Kindly resolve conflicts.

**Comment by devansh-webkul** [comment]
Confliction Resolved

**Pull request merged**